_assets/js/expand-sidenav.js

```diff
@@ -0,0 +1,32 @@
+const sidenav = () => {
+    const expandButton = document.getElementById('expand-all');
+
+    function expandAll(e) {
+        const submenuButtons = document.getElementsByClassName('usa-accordion__button');
+        if (e.target.innerText === 'Expand all') {
+            Array.from(submenuButtons).forEach(button => {
+                const isExpanded = button.getAttribute('aria-expanded') === "true";
+                if (!isExpanded) {
+                    button.click();
+                }
+            });
+            e.target.innerText = 'Collapse all';
+        } else {
+            Array.from(submenuButtons).forEach(button => {
+                const isExpanded = button.getAttribute('aria-expanded') === "true";
+                if (isExpanded) {
+                    button.click();
+                }
+            });
+            e.target.innerText = 'Expand all';
+        }
```

Comment:
Suggestion: It looks like you can [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up a bit:

```suggestion
        const submenuButtons = document.getElementsByClassName('usa-accordion__button');
        expandingAll = e.target.innerText === 'Expand all';
        Array.from(submenuButtons).forEach(button => {
            const isExpanded = button.getAttribute('aria-expanded') === "true";
            if (!isExpanded && expandingAll) button.click();
            if (isExpanded && !expandingAll) button.click();
        });
        e.target.innerText = expandingAll ? 'Collapse all' : 'Expand all';
```